be/src/exec/arrow_to_starrocks_converter.cpp

```diff
@@ -85,11 +85,16 @@ size_t fill_null_column(const arrow::Array* array, size_t array_start_idx, size_
 }
 
 void fill_filter(const arrow::Array* array, size_t array_start_idx, size_t num_elements, Filter* filter,
-                 size_t column_start_idx) {
+                 size_t column_start_idx, ArrowConvertContext* ctx) {
     DCHECK_EQ(filter->size(), column_start_idx + num_elements);
     auto* filter_data = (&filter->front()) + column_start_idx;
+    bool all_invalid = true;
     for (size_t i = 0; i < num_elements; ++i) {
         filter_data[i] = array->IsValid(array_start_idx + i);
+        all_invalid &= filter_data[i];
+    }
+    if (UNLIKELY(!all_invalid)) {
+        ctx->report_error_message("column type is not null but data is null", "");
```

Comment:
It's better to output the column name

Reply:
column name will print in `report_error_message`
```
void ArrowConvertContext::report_error_message(const std::string& reason, const std::string& raw_data) {
    if (state == nullptr) return;
    if (error_message_counter > MAX_ERROR_MESSAGE_COUNTER) return;
    error_message_counter += 1;
    std::string error_msg =
            strings::Substitute("file = $0, column = $1, raw data = $2", current_file,
                                (current_slot == nullptr) ? "null" : current_slot->col_name(), raw_data);
    state->append_error_msg_to_file(error_msg, reason);
}
```